src/components/footer.js

```diff
@@ -6,18 +6,62 @@ import styled from "styled-components"
 import { InnerWrapper } from "./layout"
 import { SlantedBorderElement } from "../common/slanted-borders-helpers"
 import colors from "../common/colors"
+import breakpoints from "../common/breakpoints"
 
 const FooterElement = styled(SlantedBorderElement)`
   background: ${colors.backgrounds.footer};
+  padding-bottom: 15px;
+  padding-top: 62px;
+  margin-top: -20px;
 `
 
-const Footer = ({ siteTitle }) => (
+const PageElements = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+`
+
+const LinkTitle = styled(Link)`
+  text-decoration: none;
+  color: black;
```

Comment:
I think we might want to use the colors.typography.dark color here. Then our theme could update that and have it take effect in the whole app.

This goes for other usages of black for text color. I think we also have a bg.dark to use as well if needed, but I haven't double checked that./

Reply:
The typography has now been fixed.